tests/test_plugin_starlette.py

```diff
@@ -365,7 +365,7 @@ def test_starlette_no_response(client):
     resp = client.get("/api/no_response")
     assert resp.status_code == 200, resp.text
 
-    resp = client.post("/api/no_response", json={"name": "starlette", "limit": 1})
+    resp = client.post("/api/no_response", json={"name": "starlette", "limit": "1"})
```

Comment:
why has this changed?

Reply:
Yeah this one was weird. So `/api/no_response` has this decorator `@api.validate(json=StrDict, resp=Response(HTTP_200=None),)` and `StrDict` model has annotation `root: Dict[str, str]`. With v1, when we passed `"limit": 1`, `parse_obj` would internally coerce that into a string, but now `model_validate` is more strict about coercing it seems like